scripts/popup.js

```diff
@@ -61,3 +73,21 @@ document.getElementById("reset-settings").addEventListener("click", () => {
   });
   window.close();
 });
+
+document.addEventListener("DOMContentLoaded", () => {
+  const hideDeleteToggle = document.getElementById("hide-delete-toggle");
+
+  hideDeleteToggle.addEventListener("change", () => {
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+      if (hideDeleteToggle.checked) {
+        chrome.tabs.sendMessage(tabs[0].id, {
+          action: "set-delete-elements",
+        });
+      } else {
+        chrome.tabs.sendMessage(tabs[0].id, {
+          action: "set-hide-elements",
+        });
+      }
```

Comment:
```suggestion
      chrome.tabs.sendMessage(tabs[0].id, {
        action: hideDeleteToggle.checked ? "set-delete-elements" : "set-hide-elements",
      });
```

Reply:
I'll check this suggestions more thoroughly later. Still, thanks for the help! I'm new to JS so the feedback really helps! :)